common/structs.go

```diff
@@ -24,4 +24,5 @@ type RunArgs struct {
 	PodCidr    string
 	SvcCidr    string
 	Provider   string
+	CustomEnv  []string
```

Comment:
I’m curious about why this structure `RunArgs` defined in common? it should in apply module.

Reply:
My understanding is that the execution command, for example, sealer run image -e port=6644 -m xxx -n xxx -p xxx should be in the run parameter, and the run parameter is in the common module.